Api/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.9"
```

Comment:
this file has to be at the same height as the API, not inside

Reply:
Sure, I will do it now